app/_locales/en/messages.json

```diff
@@ -3591,6 +3591,18 @@
     "message": "Private key for $1",
     "description": "$1 represents the account name"
   },
+  "privateKeyHidden": {
+    "message": "The private key is hidden",
+    "description": "Explains that the private key input is hidden"
+  },
+  "privateKeyShow": {
+    "message": "Show/Hide the private key input",
+    "description": "Describes a toggle that is used to show or hide the private key input"
+  },
+  "privateKeyShown": {
+    "message": "This private key is being shown",
+    "description": "Explains that the private key input is being shown"
+  },
```

Comment:
Hey @coreyjanssen, could you please check the content here some of it is for the aria label for screen readers

Reply:
Looks good to me!